account_asset_management/report/account_asset_report_xls.py

```diff
@@ -127,6 +129,29 @@ def _get_asset_template(self):
                 },
                 "width": 18,
             },
+            "purchase_value": {
+                "header": {
+                    "type": "string",
+                    "value": self._("Purchase Value"),
```

Comment:
```suggestion
                    "value": self._("Purchase Value"),
```
Why this module add self._("....") ?